test/integration/features/addons/addon-registry-route.feature

```diff
@@ -18,21 +18,34 @@ so that user can able to push their container images directly to registry and de
       Add-on 'registry-route' created docker-registry route.
       """
 
-  Scenario: As a user, I can login to OpenShift integrated docker registry
-    Given Minishift has state "Running"
-      And executing "minishift docker-env" in host shell succeeds
-      And evaluating stdout of the previous command in host shell
-      And executing "minishift oc-env" in host shell succeeds
-      And evaluating stdout of the previous command in host shell
+  Scenario: User waits until registry is fully started and accessible
      Then with up to "20" retries with wait period of "3" seconds container name "registry_docker-registry" should be "running"
-      And with up to "60" retries with wait period of "1" second command "curl -ik https://docker-registry-default.$(minishift ip).nip.io" output should contain "200 OK"
-      And executing "docker login -u developer -p `oc whoami -t` docker-registry-default.$(minishift ip).nip.io" in host shell succeeds
-      And stdout of host shell should contain "Login Succeeded"
+      And with up to "60" retries with wait period of "1000ms" the "status code" of HTTP request to "/" of service "docker-registry" in namespace "default" is equal to "200"
```

Comment:
@LalatenduMohanty I agree it looks terrible, which is mainly caused because wa have like 8 parameters in that sentence. Recently I was wondering whether we could make the timeout formatting shorter/better sounding: `with up to "X" retries with wait period of "Y"ms` as this appears in variety of steps... Maybe @robin-owen, could help us here, once he is alright :)

Reply:
Maybe we could create an issue for re-formulation of our steps, native speaker could probably find better formulations all around the test suite.

Reply:
However the step explains the whole operation in detail and ofcourse is more talkative. IMO any changes in the sentence will probably change the real sense, anyways let see how @robin-owen responds it back